HW2/P2/runtimes.txt

```diff
@@ -0,0 +1,47 @@
+N = 5:
```

Comment:
I would mention a file like this in the P2.txt file so the graders know that it exists